lib/BridgeAbstract.php

```diff
@@ -8,13 +8,26 @@
 	const MAINTAINER = 'No maintainer';
 	const CACHE_TIMEOUT = 3600;
 	const PARAMETERS = array();
+	const FORMATS = array(); // array of compatibles formats or null
```

Comment:
I think to formats like OPDS, OPML, OpenSearch Responses and eventually, iCal etc.
I would like to use the current RSS-Bridge as a start to make an universal bridges manager.
And finally, a meta search engine. a search engine that retrieve suggestions by open search plugins and customs search bridges.